### PR TITLE
Add terminology bindings to DSTU 2 documentation for Condition resource.

### PR DESCRIPTION
**Pull request by naveenaluru**
Adding terminology table for condition resource

### CODE REVIEW

File: lib/resources/dstu2/condition.yaml [hunk 1]
```diff
@@ -71,6 +72,15 @@ fields:
                    "display": "Generalized nonconvulsive epilepsy",
                    "userSelected": true}],
                 "text": "Absence seizures"}}
+  binding:
+    description: A code that defines the condition.
```
Comment (cerbs) [inline lib/resources/dstu2/condition.yaml]: These descriptions should probably describe the binding more than the value type. In other words, they should be closer to the values in the FHIR terminology table: http://hl7.org/fhir/DSTU2/condition.html#tx. In this case a lot of the terminology descriptions are the same, or nearly the same, as the field descriptions

File: lib/resources/dstu2/condition.yaml [hunk 2]
```diff
@@ -71,6 +72,15 @@ fields:
...
+    terminology:
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+    - display: ICD-9
+      system: http://hl7.org/fhir/sid/icd-9
```
Comment (cerbs) [inline lib/resources/dstu2/condition.yaml]: Is this the correct system for ICD-9?  The ICD-10 system url redirects to a Terminologies-systems page which lists all of the valid systems including that specific ICD-10 system, however it doesn't list this specific ICD-9 system. Instead it lists http://hl7.org/fhir/sid/icd-9-cm.  'http://hl7.org/fhir/sid/icd-9' does link to a FHIR page, but it looks like it is a working/temporary page. The page title literally has 'C:\work\org.hl7.fhir\...' in it.

File: lib/resources/dstu2/condition.yaml (hunk 1 above)
Comment (naveenaluru) [inline lib/resources/dstu2/condition.yaml]: Done.

**Comment by cerbs** [comment]
:+1:

File: lib/resources/dstu2/condition.yaml (hunk 2 above)
Comment (naveenaluru) [inline lib/resources/dstu2/condition.yaml]: Agreed that 'http://hl7.org/fhir/sid/icd-9' is not listed in terminologies system. But this is currently what we are returning in response and the fhir page it links to provides source of this code system.

**Comment by davischristi** [comment]
:+1:

**Pull request merged**